### ISSUE TITLE
Remove aria-busy and add hidden text for the spinner component

### ISSUE THREAD

**Issue by Veyfeyken** [a-ui/core_branding_scss#147]
### Problem

[atoms/atoms.spinner](https://a-ui.github.io/core_branding_scss/#atoms-spinner)

1. The spinner component contains an aria-busy attribute which is not appropriate in this context.  

 The aria-busy state indicates an element is being modified and that assistive technologies may want to wait until the modifications are complete before exposing them to the user. It should be placed on the container element wich owns the content that is being modified.

 ```html
<div class="a-spinner" role="alert" aria-busy="true"></div>
```

2. The ARIA role="alert" creates a live region in order to 'alert' screenreaders of the change. But the div currently has no content so there is no information for screenreaders to announce.

### Solution

* Remove the aria-busy attribute.
* Add hidden text for screenreaders

Suggestion:

```html
<div class="a-spinner" role="alert">
 <span class="u-screen-reader-only">loading</span>
</div>
```

### Links and articles
* [AnySurfer: Live regions (Dutch)](https://www.anysurfer.be/nl/documentatie/artikels/detail/live-regions)
* [WAI-ARIA 1.2: aria-busy (state)](https://www.w3.org/TR/wai-aria-1.2/#aria-busy)

**Comment by TriangleJuice** [comment]
Fixed in #178
